pkg/boot/kexec/memory_linux.go

```diff
@@ -667,3 +667,52 @@ func (m *MemoryMap) Insert(r TypedRange) {
 	newMap.sort()
 	*m = newMap
 }
+
+// Memory type (uint32) in UefiPayload
+const (
+	PayloadTypeRAM      = 1
+	PayloadTypeDefault  = 2
+	PayloadTypeAcpi     = 3
+	PayloadTypeNVS      = 4
+	PayloadTypeReserved = 5
+)
+
+// payloadMemoryMapEntry represent a memory map entry in payload param
+type payloadMemoryMapEntry struct {
+	Start uint64
+	End   uint64
+	Type  uint32
```

Comment:
I'd make this a concrete type, and assign that type to the consts, even though we are going to refactor later.

Reply:
Done, but I am not sure if this is what you mean. PTAL